independent-publisher-connectors/Safaricom Mpesa Daraja/apiProperties.json

```diff
@@ -0,0 +1,49 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "4b048dda-9dc5-412b-b881-1f6881f882d8",
```

Comment:
@johnmuchiriM and @Bkaraba secret look alike values are not allowed. please replace the client ID with a dummy value. You can use something like "Enter your client ID"